website/content/en/highlights/2022-04-06-disk-buffer-v2-stable.md

```diff
@@ -0,0 +1,64 @@
+---
+date: "2022-04-06"
+title: "Switching the default implementation of disk buffers to `disk_v2`"
+description: "After much testing, we're promoting `disk_v2` to stable to bring performance and efficiency benefits to everyone using disk buffers."
+authors: ["tobz"]
+pr_numbers: [12069]
+release: "0.22.0"
+hide_on_release_notes: false
+badges:
+  type: enhancement
+---
+
+Back in February, we announced the beta release of our new, reworked disk buffer implementation --
```

Comment:
👍 nice write up. I created an issue to remind me to call attention to it in the release description too https://github.com/vectordotdev/vector/issues/12121

Reply:
🙇🏻